index.html

```diff
@@ -4,18 +4,169 @@
 <head>
     <meta charset="UTF-8">
 
-    <title>Untitled</title>
+    <title>Smashing Magazine - HeatMap</title>
 
     <link rel="stylesheet" href="css/styles.css">
 
+    <link href="https://fonts.googleapis.com/css?family=Fira+Sans:200,500,700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Playfair+Display:400,700&display=swap" rel="stylesheet">
+
 </head>
 
 <body>
     <header>
```

Comment:
### While this is semantically correct let's try to include the `<nav>`  tag with seven `<div>` tags instead of five, as there are seven elements on the original site, and make them span across the whole width and the whole height of the header. element.